merlin/models/tf/__init__.py

```diff
@@ -181,6 +180,7 @@
     "AvgPrecisionAt",
     "RecallAt",
     "BaseModel",
+    "PredictionContext",
```

Comment:
If this is intended to be an internal implementation detail, then maybe we should remove from this top-level export?